src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -342,6 +445,14 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, OrgPersonMix
     return this.getOrgPeople
   }
 
+  /** Whether to show table headers. */
+  get showHeaders (): boolean {
+    // true if there are any org-persons to display
+    // include REMOVED as per currentPeopleAndRoles above
+    // exclude replaced-removed
+    return this.currentPeopleAndRoles.some(orgPerson => !(this.wasReplaced(orgPerson) && this.wasRemoved(orgPerson)))
+  }
```

Comment:
We need this to hide the table headers if no org-persons are showing, for example while replacing a SP's lone proprietor.